src/main.rs

```diff
@@ -196,6 +196,20 @@ struct CLI {
     )]
     file_path: Option<PathBuf>,
 
+    #[structopt(
+        name = "negative-file",
```

Comment:
I would rather call this option `exclusions-file`. This conveys the meaning more clearly, in my opinion. But this is something that I can change myself after merging in case you don't want to.

Reply:
done.